src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -65,9 +77,9 @@
                                             MeterName="@(PageViewModel.SelectedMeter.MeterName)"
                                             InstrumentName="@(PageViewModel.SelectedInstrument.Name)"
                                             Duration="PageViewModel.SelectedDuration.Id"
-                                            ActiveView="@(PageViewModel.SelectedViewKind ?? MetricViewKind.Graph)"
+                                            ActiveView="@(PageViewModel.SelectedViewKind ?? Metrics.MetricViewKind.Graph)"
                                             OnViewChangedAsync="@OnViewChangedAsync"
-                                            Applications="_applications" />
+                                            Applications="@_applications" />
                                     }
                                     else if (PageViewModel.SelectedMeter != null)
                                     {
```

Comment:
My change was lost during a merge:

```
<h3 class="meter-name-title">@PageViewModel.SelectedMeter.MeterName</h3>
```

Reply:
I don't understand when it was removed or why the diff doesn't show anything, re-added.